cli/cmd/root.go

```diff
@@ -42,6 +42,11 @@ func NewCmdRoot() *cobra.Command {
 	noLogFile := false
 	logFileName := "cloudquery.log"
 	sentryDsn := sentryDsnDefault
+	var err error
+	if invocationUUID.UUID, err = guuid.NewRandom(); err != nil {
+		fmt.Fprintf(os.Stderr, "failed to generate invocation uuid: %s", err.Error())
```

Comment:
```suggestion
		fmt.Fprintf(os.Stderr, "failed to generate invocation uuid: %v", err)
```

Reply:
fixed in [217cc13](https://github.com/cloudquery/cloudquery/pull/18878/commits/217cc1309b098c279c07aa4695cc7ca1ebd2eb0e)